pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -157,6 +157,9 @@ type ClusterSpec struct {
 	// +optional
 	WaitTimeoutForHealthyOSDInMinutes time.Duration `json:"waitTimeoutForHealthyOSDInMinutes,omitempty"`
 
+	// waitTimeoutForCephCommandInSeconds defines the timeout (in seconds) of Ceph CLI commands
+	WaitTimeoutForCephCommandInSeconds int `json:"waitTimeoutForCephCommandInSeconds,omitempty"`
```

Comment:
Instead of adding this setting in the cluster CRD, perhaps it makes sense to make it an env var on the operator deployment. It seems more related to the operator deployment, but it could also be a cluster setting if you wanted a different timeout per cluster. I could go either way, it doesn't seem critical.

Reply:
I'm a bit hesitant to have this in the CR too. IMO it's an internal mechanism that should not be surfaced to the user. How will/should the user know about this timeout? If there is an internal failure, we should retry or go with a higher timeout maybe?

Reply:
@travisn @leseb 

I'd like to add an option to specify the timeout value in `rook-ceph-operator-config` cm, as Travis said.

It's because it's really hard to estimate the appropriate timeout and too long timeout is practically useless. In addition, the cost and the effect of implementing retry logics would be big since there are plenty of Ceph commands execution. I consider it's overkill just for this purpose.

Reply:
I think it makes sense to use the `rook-ceph-operator-config` to tweak the operator internals.